src/services/SpeciesService.ts

```diff
@@ -330,6 +330,62 @@ const suggestSpecies = async (organizationId: number, query: string): Promise<Su
   return (await SearchService.search(params)) as SuggestedSpecies[] | null;
 };
 
+// find species that are used in the org
+const getInUseSpecies = async (organizationId: number): Promise<Record<number, string>> => {
+  type WithSpecies = { species_id: number; species_scientificName: string };
+  type AccessionFacility = { accessions: WithSpecies[] };
+  type PlantingSitePopulation = { populations: WithSpecies[] };
+
+  const params: SearchRequestPayload = {
+    prefix: '',
+    fields: [
+      'facilities.accessions.species_id',
+      'facilities.accessions.species_scientificName',
+      'batches.species_id',
+      'batches.species_scientificName',
+      'plantingSites.populations.species_id',
+      'plantingSites.populations.species_scientificName',
+    ],
```

Comment:
@sgrimm thoughts on this search query - to identify used species in an org?
This function returns a map of species id to scientific name, hence the _scientificName fields.

Reply:
Thanks, yes, there are duplicates returned. Code does some de-duping. API sounds good, l will take that approach.